From 98a0f19df0a31d5649ad89d395fd1b8de5591827 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 3 Apr 2014 21:33:25 +0800
Subject: [PATCH] admin/Makefile.am: only install configured.py once

Both the two rules install-adminpyDATA and install-generatedDATA will
install the configured.py to the same location, they can run parallel,
and they use "install -m", which would might build failures:

/usr/bin/install: setting permissions for `/path/to/configured.py': No such file or directory

This is because the first install is setting the permission while the
second install is removing the file an re-install.

Only install the configured.py once will fix the problem, I think that
there is no side effect since it installed the same file to the same
location twice in the past.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 admin/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/admin/Makefile.am b/admin/Makefile.am
index ce5937b..6f96934 100644
--- a/admin/Makefile.am
+++ b/admin/Makefile.am
@@ -73,7 +73,6 @@ SystemStatsWidgets.py \
 Wizard.py \
 XMLServerDigest.py \
 config_version.py \
-configured.py \
 consts.py \
 util.py \
 popen.py \
-- 
1.8.2.1

